dockerfiles/rpm.dockerfile

```diff
@@ -73,10 +73,10 @@ ENV CC=gcc
 WORKDIR /root/rpmbuild
 
 # Install build dependencies and build scripts
-COPY --from=go-md2man /go/bin/go-md2man /go/bin/go-md2man
-COPY rpm/containerd.spec SPECS/containerd.spec
-COPY scripts/build-rpm    /root/
-COPY scripts/.rpm-helpers /root/
+COPY --link --from=go-md2man /go/bin/go-md2man /go/bin/go-md2man
+COPY --link rpm/containerd.spec SPECS/containerd.spec
+COPY --link scripts/build-rpm    /root/
+COPY --link scripts/.rpm-helpers /root/
```

Comment:
Same here; I expect individual files to be "merged" with anything that could be present inside `/root/`